src/core/ext/filters/logging/logging_filter.cc

```diff
@@ -257,6 +257,7 @@ class CallData {
                        const ServerMetadata* metadata) {
     LoggingSink::Entry entry;
     if (metadata != nullptr) {
+      entry.is_trailer_only = metadata->get(GrpcTrailersOnly()).value_or(false);
```

Comment:
If there is a server header, it cannot be trailers only, so this seems redundant?